app/api/route.js

```diff
@@ -0,0 +1,25 @@
+import { createClient } from "@supabase/supabase-js";
+import { NextResponse } from "next/server";
+
+export const supabase = createClient(
+  "https://yygevqkucilegrhzrslh.supabase.co",
+  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Inl5Z2V2cWt1Y2lsZWdyaHpyc2xoIiwicm9sZSI6ImFub24iLCJpYXQiOjE2OTA4MTk3NjAsImV4cCI6MjAwNjM5NTc2MH0.ZWQ0xwnlvDrxe1GBUyqCJ_3UsbwAFQghg2-m-Ji3esU"
+);
+
+export default async function handler(req, res) {
```

Comment:
Este metodo deberia tener otro nombre! Fíjate lo que tenes en el TODO APP.
```suggestion
export default async function handler(req, res) {
```

Reply:
Te esta tirando un `404` Not Found. Eso significa que la ruta `api/route` no tiene implementado el metodo `GET`.

Fijate como lo tenes el [TODO APP](https://github.com/SG-SantiagoGilgorri/ToDo-App/blob/e2379e45d2a695701b74cbfaad2aaa88cbe40539/app/api/route.js#L21)!